app/src/main/java/com/framgia/englishconversation/screen/audiodetail/AudioDetailViewModel.java

```diff
@@ -87,6 +87,11 @@ public boolean onItemLongClick(View viewGroup, TimelineModel item) {
                     return false;
                 }
 
+                @Override
```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 16, expected level should be one of the following: 8, 12.